src/main/java/baron/database/Database.java

```diff
@@ -0,0 +1,148 @@
+package baron.database;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import baron.utils.FileUtils;
+import baron.utils.StringUtils;
+
+/**
+ * Handles CRUD operations for a given file path, create, read, update and delete
+ */
+public class Database {
+
+    // Base directory for the files to be read from. Defaults to root directory
+    public static final String BASE_DIR = System.getProperty("user.dir");
+
+    /**
+     * Gets the File object from the BASE_DIR with the specified fileName.
+     * Creates the file if it doesn't exist
+     *
+     * @param fileName File name of the file to read from
+     * @return the file object of the table
+     */
+    public static File getTable(String fileName) {
+        try {
+            String filePath = BASE_DIR + "/" + fileName + ".txt";
+            File db = new File(filePath);
+            if (!db.exists() && db.createNewFile()) {
+                System.out.println("Creating new file");
+            }
+            return db;
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+        return null;
+    }
+
+    /**
+     * Appends a new line to the end of the file
+     *
+     * @param filePath Filepath to write
+     * @param line     Line to append to the end of file
+     * @return the id of the newly appended line, which is actually just lineNumber - 1
+     */
+    public static long create(Path filePath, String line) {
```

Comment:
Perhaps we could use a more appropriate name for this method, such as write/writeToFile()?